doc/api.yml

```diff
@@ -3825,6 +3859,19 @@ components:
           $ref: '#/components/schemas/FileType'
         file_url:
           type: string
+    ShareInfo:
+      type: object
+      properties:
+        expiration:
+          type: string
+        name:
+          type: string
+        path:
+          type: string
+        url:
+          type: string
+        permissions:
+          type: string
```

Comment:
permissions ? means it's plural ? 
if so, it has to be an array of permissions.

If permissions are known already known, you should use an ENUM